styles.css

```diff
@@ -132,22 +143,34 @@ a:visited {
   font-weight: 400;
   font-size: 40px;
   color: #1c1a19;
-  border-bottom: 1px solid #989898;
-  padding-bottom: 40px;
-  margin-bottom: 80px;
+  padding-bottom: 25px;
+}
+
+.works-line {
+  width: 100%;
+  height: 1px;
+  max-width: 770px;
+  min-width: 327px;
 }
 
 .works-image1 {
-  max-width: 327px;
-  margin-bottom: 24px;
+  width: 327px;
```

Comment:
- [ ] Kindly make sure this part's flex direction is row above 768px, so that your design could match the figma's.

  - Your design:
![YourRecent](https://user-images.githubusercontent.com/97834160/206579946-2a0e487c-6ee7-4938-a72f-b7b3e1341178.png)

  - Figma design:
![figRecent](https://user-images.githubusercontent.com/97834160/206580006-087145e9-bd5f-4a04-82f5-eb99a249db60.png)